src/Cli/dotnet/Commands/CliCommandStrings.resx

```diff
@@ -1619,6 +1619,9 @@ Tool '{1}' (version '{2}') was successfully installed. Entry is added to the man
     <value>Duplicate directives are not supported: {0}</value>
     <comment>{0} is the directive type and name.</comment>
   </data>
+  <data name="QuoteInDirective" xml:space="preserve">
+    <value>Directives currently cannot contain quotes (").</value>
```

Comment:
So, is this specific to double quotes `"` or does that also account for single quotes `'`? Might make sense to be a bit more specific.

Reply:
Currently this is specific to double quotes, I will clarify, thanks. We could disallow single quotes too, I didn't think about that, but it also seems unnecessary (we likely won't want to support single-quoted directive values anyway, only double-quoted, I imagine, same as C# strings).